src/eko/evolution_operator/__init__.py

```diff
@@ -40,8 +42,8 @@ def select_singlet_element(ker, mode):
         ker : complex
             singlet integration kernel element
     """
-    k = 0 if mode[2] == "q" else 1
-    l = 0 if mode[3] == "q" else 1
```

Comment:
Write explicitly this encoding in the docstring, and do it everywhere, otherwise it will be impossible to debug in a few weeks/months.